test/controllers/organization_memberships_controller_test.exs

```diff
@@ -110,18 +110,15 @@ defmodule CodeCorps.OrganizationMembershipControllerTest do
     end
 
     test "renders page not found when id is nonexistent", %{conn: conn} do
-      assert_error_sent 404, fn ->
-        path = conn |> organization_membership_path(:show, -1)
-        conn |> get(path)
-      end
+      path = conn |> organization_membership_path(:show, -1)
+      assert conn |> get(path) |> json_response(:not_found)
```

Comment:
These actually weren't rendering json, due to not using Canary's `load_resource`, which would then trigger a not-found handler.